### DISCUSSION TITLE
Export notifications not being send

### DISCUSSION THREAD

**Discussion by JulieVelghe** [solspace/craft-freeform#1325] [Questions & Answers] [answered]
I have a profile setup on a site that gets submissions from a certain form from the last 7 days. I want to mail an export from that profile.
For test purposes, I have set it up to try and send it daily.
![image](https://github.com/solspace/craft-freeform/assets/46524074/693bec65-427e-46b3-af0d-75fc5b1a2b78)

However, I don't get any notification mails. I've looked in the log files, but don't find any errors. Is there a database table that could give more clarity as to why there are no mails?

I'm on craft 4.9.5 and Freeform 5.3.1.

Any insight is very much appreciated.

**Comment by JulieVelghe** [comment]
Hello @kjmartens , could you perhaps take a look at this issue?
Thanks in advance for the effort :)

**Comment by kjmartens** [comment]
Sorry for the delay @JulieVelghe,

What happens if you enter a value (anything, e.g. `Hello`) inside the **Email Message** textarea? Does it make any difference? 🙂 

Can you also share a screenshot of what the Export Profile looks like?

**Comment by JulieVelghe** [comment]
Hi @kjmartens , unfortunately, filling in the email message doesn't make any difference :( 
I'll add a picture of the export profile
![Aanvragen-stages-last-7-days-i-mens-be-Stage-aanvragen-i-mens](https://github.com/solspace/craft-freeform/assets/46524074/e9d3de66-5514-4c7e-b61d-2fbe35e40113)

**Reply by JulieVelghe** [reply]
I don't know if this has any influence, but we are using sendgrid in our mailing settings.

**Reply by JulieVelghe** [reply]
@kjmartens I think I may have found the cause. The emailadress settings in craft are using the following emailadres as sender: admin@cabinetc.be
I've seen in the log this morning that there was an error 'Unable to send an email: the from address does not match a verified Sender Identity'. So I'm assuming that these notifications use the emailsettings from Craft.
Is there a way to go around this?  In the email twig templates that are used for other notifications, we override this with code:

```
{# subject: Nieuwe formulier inzending #}
{# fromEmail: info@i-mens.be #}
{# fromName: i-mens #}
{# replyToEmail: info@i-mens.be #}
{# cc: #}
{# bcc: #}
{# includeAttachments: true #}
{# presetAssets: #}
{# description: Admin e-mail: details van een formulier inzending #}
```
Is there a way to do something similar for the export notifications?

**Reply by kjmartens** [reply, marked as answer]
This is expected behavior when using SMTP -- the "from email address" should match what is set for the Craft email settings. It's not something Freeform or other plugins have control over unfortunately.

The best you can do is set the `replyToEmail` to whatever you want it to be, and at least anyone replying back would have the email of your choice used. 🙂
